File: C#/백준/Bronze/31090. 2023은 무엇이 특별할까？/2023은 무엇이 특별할까？.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0; i<a; i++){
                int b = int.Parse(Console.ReadLine());
                int c = b%100;
                if((b+1)%c==0){
                    Console.WriteLine("Good");
                }
                else{
                    Console.WriteLine("Bye");
                }
            }
        }
    }
}

File: Java/백준/Bronze/31090. 2023은 무엇이 특별할까？/2023은 무엇이 특별할까？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i=0; i<a; i++){
            int b = in.nextInt();
            int c = b%100;
            if((b+1)%c==0){
                System.out.println("Good");
            }
            else{
                System.out.println("Bye");
            }
        }
    }
}